.gitignore

```diff
@@ -1 +1,2 @@
 /.idea/
+actual
```

Comment:
The test outputs this folder if it fails for easy comparison and updating of the expected values. It should never be checked in so I figured it should be added to project level gitignore. 

I can scope it down to be more specific if desired.

Reply:
I have no issue with the line being there, however it should be
```
/actual
```
so that it isn't applied throughout the entire directory structure. Just we aren't generating any runtime data for `actual` as far as I can tell so it might have been better in the `.git/info/exclude` file (which is per repo and local only.)

Reply:
scoped this more narrowly, since this does create files during runtime (only when running serialization test & only if there are failures)